#include "Output.apb.h"
bool to_pb_v2(COutput &apb, std::string &dest);
#include "Payment.apb.h"
bool to_pb_v2(CPayment &apb, std::string &dest);
#include "PaymentAck.apb.h"
bool to_pb_v2(CPaymentAck &apb, std::string &dest);
#include "PaymentDetails.apb.h"
bool to_pb_v2(CPaymentDetails &apb, std::string &dest);
#include "PaymentRequest.apb.h"
bool to_pb_v2(CPaymentRequest &apb, std::string &dest);
#include "Output.apb.h"
bool to_pb_v3(COutput &apb, std::string &dest);
#include "Payment.apb.h"
bool to_pb_v3(CPayment &apb, std::string &dest);
#include "PaymentAck.apb.h"
bool to_pb_v3(CPaymentAck &apb, std::string &dest);
#include "PaymentDetails.apb.h"
bool to_pb_v3(CPaymentDetails &apb, std::string &dest);
#include "PaymentRequest.apb.h"
bool to_pb_v3(CPaymentRequest &apb, std::string &dest);
